helm-charts/blog/templates/NOTES.txt

```diff
@@ -0,0 +1,11 @@
+
+Access the deployed blog in the following URL:
+
+{{- if eq true .Values.ingress.tls }}
+      https://{{ .Values.ingress.hostname}}
+{{- else }}
+      http://{{ .Values.ingress.hostname}}
+{{- end }}
+
+NOTE: It may take a few minutes for the Ingress IP to be available.
+      Watch the status with: 'kubectl get ingress {{ template "fullname" . }}'
```

Comment:
you need to use `describe` rather than `get` to see the status with all the needed detail